### DISCUSSION TITLE
Automatic selected servers from criterias

### DISCUSSION THREAD

**Discussion by Kylvan-8** [DNSCrypt/dnscrypt-proxy#1932] [Q&A] [answered]
Hello there,

I would like to know how to setup the toml configuration file to let dnsec choose automatically the server depending of the criterias of this same file.

max_clients = 250

ipv4_servers = true

ipv6_servers = true

dnscrypt_servers = true

doh_servers = true

require_dnssec = true

require_nolog = true

require_nofilter = true

Regards

**Comment by welwood08** [comment, marked as answer]
The syntax of that line is invalid because it is truncated. It looks like it was opened in a terminal text editor that was not wide enough to display the whole line and then copied into the comment.

A quick fix would be to replace `,'sdns://gRE>` at the end with `] }`.

**Reply by Kylvan-8** [reply]
Thank you i got it while you was responding, got another issue after that. Do i have to put stricted dns servers in " servers_names"  or dnsec could choose automatically one from criterias ?

**Reply by Kylvan-8** [reply]
I solved it by adding # before servers_name. Thank for your help anyway !

**Reply by welwood08** [reply]
To quote the example config:
```
## By default, this list is empty and all registered servers matching the
## require_* filters will be used instead.
```
